### DISCUSSION TITLE
Blinking cursor?

### DISCUSSION THREAD

**Discussion by 9mm** [neovide/neovide#2160] [Q&A]
Ive tried all the different variations i can think of with:

```
vim.opt.guicursor = "a:blinkon100"
```

but i cant seem to find how to get a block cursor in insert mode that blinks

do i need to do something extra?

**Comment by fredizzimo** [comment]
It does appear to be broken, the last time I knew it was working was when I changed the code for the rendering branch, so something must have broke after that. Can you open an issue?
